### PR TITLE
Add test for attaching ML file

### PR DESCRIPTION
**Pull request by olukas**
8 hours soak tests run for this test passed. It executes ~6000 job during 8 hours.

### CODE REVIEW

File: attach-ml-model-test/src/main/java/com/hazelcast/jet/tests/attachml/AttachMLModelTest.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    }
+
+    @Override
+    protected void init() throws Exception {
+        mlDataPath = property("mlDataPath", ML_DATA_PATH_DEFAULT);
+    }
+
+    @Override
+    protected void test() throws IOException {
+        Path mlData = new File(mlDataPath).toPath();
```
Comment (gurbuzali) [inline attach-ml-model-test/src/main/java/com/hazelcast/jet/tests/attachml/AttachMLModelTest.java]: you can use `Paths.get(mlDataPath)`

File: attach-ml-model-test/src/main/java/com/hazelcast/jet/tests/attachml/AttachMLModelTest.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        String firstLine = Files.lines(mlData).findFirst().get();
+        assertEquals(EXPECTED_FIRST_LINE_IN_MODEL, firstLine);
+
+        long begin = System.currentTimeMillis();
+        long jobCount = 0;
+        while (System.currentTimeMillis() - begin < durationInMillis) {
+            JobConfig jobConfig = new JobConfig();
+            jobConfig.attachFile(mlDataPath, MODEL + jobCount);
+            jet.newJob(pipeline(jobCount), jobConfig).join();
+            if (jobCount % LOG_JOB_COUNT_THRESHOLD == 0) {
```
Comment (gurbuzali) [inline attach-ml-model-test/src/main/java/com/hazelcast/jet/tests/attachml/AttachMLModelTest.java]: how frequently we will log this ? the job will be very short I guess

File: attach-ml-model-test/src/main/java/com/hazelcast/jet/tests/attachml/AttachMLModelTest.java (hunk 1 above)
Comment (olukas) [reply attach-ml-model-test/src/main/java/com/hazelcast/jet/tests/attachml/AttachMLModelTest.java]: I'll fix it :+1:

File: attach-ml-model-test/src/main/java/com/hazelcast/jet/tests/attachml/AttachMLModelTest.java (hunk 2 above)
Comment (olukas) [reply attach-ml-model-test/src/main/java/com/hazelcast/jet/tests/attachml/AttachMLModelTest.java]: It logs every ~8minutes. If I ran it it in AWS it executed ~6000 job during 8 hours run. Locally (when all nodes are on the same machine and only this test is running) it is much faster - it logs every ~10seconds.

**Pull request merged**